vm/systemSmartContracts/esdt.go

```diff
@@ -378,7 +378,7 @@ func (e *esdt) registerNonFungible(args *vmcommon.ContractCallInput) vmcommon.Re
 	}
 
 	tokenType := []byte(core.NonFungibleESDT)
-	if e.enableEpochsHandler.DynamicESDTEnabled() {
+	if e.enableEpochsHandler.IsFlagEnabled(common.DynamicESDTFlag) {
```

Comment:
since we have merged the PR with the rc/v1.7.0, please add the `common.DynamicESDTFlag` to the list on L126